build-system/global-configs/prod-config.json

```diff
@@ -29,6 +29,7 @@
   "ios-fixed-no-transfer": 0,
   "layoutbox-invalidate-on-scroll": 1,
   "pump-early-frame": 1,
+  "random-subdomains-for-safeframe": 1,
```

Comment:
This would enable the AMP experiment for 100% of traffic with 50% in control and 50% in experiment branches.  Instead let's limit to 0.1 (5% each branch).  Also should enable in canary branch (canary-config.json).

Reply:
done

Reply:
done